package.json

```diff
@@ -10,6 +10,7 @@
     "lint:fix:sol": "prettier --write '{packages,examples}/*/{contracts,src}/**/*.sol'",
     "lint:fix": "pnpm run '/^lint:fix:(js|sol)/'",
     "build": "pnpm -r --filter @usecannon/builder --filter @usecannon/cli --filter hardhat-cannon --filter @usecannon/api run build",
+    "watch": "pnpm -r --parallel --filter @usecannon/builder --filter @usecannon/cli run watch",
```

Comment:
added a helper script to be able to just run `pnpm watch` on the root, instead of having to open several tabs